src/mscorlib/shared/System/DateTimeOffset.cs

```diff
@@ -788,13 +849,28 @@ public static Boolean TryParse(String input, IFormatProvider formatProvider, Dat
             return parsed;
         }
 
+        public static bool TryParseExact(
+            ReadOnlySpan<char> input, string format, IFormatProvider formatProvider, DateTimeStyles styles, out DateTimeOffset result)
```

Comment:
nit: I prefer either have the whole method header written in one line, or having each param in its own line.